articles/ai-services/agents/includes/quickstart-javascript.md

```diff
@@ -69,6 +69,13 @@ For example, your connection string may look something like:
 
 Set this connection string as an environment variable named `PROJECT_CONNECTION_STRING` in a `.env` file.
 
+### Security Notice 
```

Comment:
@aahill @diberry - Should the heading at line 72 be deleted? It seems unnecessary now that the newly added content is formatted as a note. It also places all of the content after line 76 under this H3 subheding.